[Bug]: Collections > Location Collections > Regions > Eastern Asia

### Describe the bug

When using
Collections > Location Collections > Regions > Eastern Asia
For some reason, South-Eastern Asia is also created even though this specific collection is not selected in the menu

### Steps to reproduce

1. Go To Collections > Location Collections > Regions > Eastern Asia
2. Let it generate collections.
3. Check in Plex, South-Eastern Asia is also created without it being selected in Quickstart

### Config source

Fresh walkthrough (Quickstart)

### How are you running Quickstart?

Docker

### Support Info Output

```shell
#==================== elated_wilson ====================#
# elated_wilson config created by Quickstart on 2026-04-16 00:29:39
# System Information
# OS: Linux 6.6.x
# Docker: True
# CPU: Unknown CPU (4 cores)
# Memory: 2442 MB / 15770 MB (15%) | 13328 MB Free
# Python: 3.13.13
# Git: Unavailable
# Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/147.0.0.0 Safari/537.36
# Quickstart Port: 7171
# Quickstart Debug: Disabled
# Quickstart Theme: kometa
# Quickstart Optimize Template Defaults: Enabled
# Quickstart Config Archive History: Keep all (0)
# Quickstart Kometa Log Retention: Keep all (0)
# Quickstart Test Libraries Temp Path: /modules/../config/tmp
# Quickstart Test Libraries Install Path: /modules/../config/plex_test_libraries
# Quickstart Session Lifetime Days: 30
# Connected to Plex server AsustorNAS version 1.43.1.10611-1e34174b1
# Running on Linux version 6.6.x
# Plex DB cache setting: 40 MB
# PlexPass: Unknown on Public update channel.
# Scheduled maintenance running between 1:00 and 4:00
# Quickstart: 0.9.16-build12 | Branch: develop | Environment: Docker
###
# Libraries configured with Quickstart: 4 movie, 4 show
# Information on library: 4K Movies
# Type: Movie
# Agent: tv.plex.agents.movie
# Scanner: Plex Movie
# Ratings Source: IMDb
# Content Count: 110 movies
#
# Information on library: Anime Movies
# Type: Movie
# Agent: tv.plex.agents.movie
# Scanner: Plex Movie
# Ratings Source: IMDb
# Content Count: 73 movies
#
# Information on library: Documentaries
# Type: Movie
# Agent: tv.plex.agents.movie
# Scanner: Plex Movie
# Ratings Source: Rotten Tomatoes
# Content Count: 67 movies
#
# Information on library: Movies
# Type: Movie
# Agent: tv.plex.agents.movie
# Scanner: Plex Movie
# Ratings Source: IMDb
# Content Count: 1696 movies
#
# Information on library: 4K TV Shows
# Type: Show
# Agent: tv.plex.agents.series
# Scanner: Plex TV Series
# Ratings Source: N/A
# Content Count: 1 shows / 6 episodes
#
# Information on library: Anime
# Type: Show
# Agent: tv.plex.agents.series
# Scanner: Plex TV Series
# Ratings Source: N/A
# Content Count: 751 shows / 22926 episodes
#
# Information on library: Docu Series
# Type: Show
# Agent: tv.plex.agents.series
# Scanner: Plex TV Series
# Ratings Source: N/A
# Content Count: 29 shows / 522 episodes
#
# Information on library: TV Shows
# Type: Show
# Agent: tv.plex.agents.series
# Scanner: Plex TV Series
# Ratings Source: N/A
# Content Count: 380 shows / 17731 episodes
###
# Quickstart log tail (last 200 lines)

[2026-04-16 08:34:49,755] [INFO]    | New log started at 2026-04-16 08:34:49.755540
[2026-04-16 08:34:51,644] [INFO]    | Running on port: 7171 | Debug Mode: Disabled
[2026-04-16 08:34:51,727] [INFO]    | Running in headless mode — no system tray will be shown...
[2026-04-16 08:34:51,727] [INFO]    | Quickstart is Running inside Docker.
[2026-04-16 08:34:51,727] [INFO]    | Access it at http://<your-server-ip>:7171
[2026-04-16 08:34:51,728] [INFO]    | Note: This IP is the HOST machine IP, not the container IP.
[2026-04-16 08:34:51,728] [INFO]    | Port and Debug Settings can be amended via the Settings cog in the UI or by editing your config/.env file
[2026-04-16 08:34:52,133] [INFO]    | Checked for updates.
[2026-04-16 22:09:06,930] [TIMING]  | [8fdac2a1] Loading attribute_config...
[2026-04-16 22:09:06,931] [TIMING]  | [8fdac2a1] Loading collection_config...
[2026-04-16 22:09:06,941] [TIMING]  | [8fdac2a1] Loading overlay_config...
[2026-04-16 22:09:06,946] [TIMING]  | [8fdac2a1] Loading quickstart_root...
[2026-04-16 22:09:06,946] [TIMING]  | [8fdac2a1] Start render_template...
[2026-04-16 22:09:07,678] [DEBUG]   | [8fdac2a1] Incoming request size: 0.04 KB
[2026-04-16 22:09:35,519] [DEBUG]   | [8fdac2a1] Incoming request size: 0.07 KB
[2026-04-16 22:09:35,520] [DEBUG]   | [8fdac2a1] Form field count: 3
[2026-04-16 22:09:36,811] [INFO]    | [8fdac2a1] db_cache returned from Plex: 40
[2026-04-16 22:09:36,997] [ERROR]   | [8fdac2a1] Error validating Plex server: (503) service_unavailable; https://plex.tv/api/v2/user <html>
 <head><title>503 Service Temporarily Unavailable</title></head>
 <body>
 <center><h1>503 Service Temporarily Unavailable</h1></center>
 <hr><center>nginx</center>
 </body>
 </html>
 
[2026-04-16 22:09:37,007] [TIMING]  | [8fdac2a1] Loading attribute_config...
[2026-04-16 22:09:37,010] [TIMING]  | [8fdac2a1] Loading collection_config...
[2026-04-16 22:09:37,023] [TIMING]  | [8fdac2a1] Loading overlay_config...
[2026-04-16 22:09:37,029] [TIMING]  | [8fdac2a1] Loading quickstart_root...
[2026-04-16 22:09:37,029] [TIMING]  | [8fdac2a1] Start render_template...
[2026-04-16 22:09:40,188] [DEBUG]   | [8fdac2a1] Incoming request size: 0.55 KB
[2026-04-16 22:09:40,189] [DEBUG]   | [8fdac2a1] Form field count: 14
[2026-04-16 22:09:41,292] [INFO]    | [8fdac2a1] db_cache returned from Plex: 40
[2026-04-16 22:09:41,460] [ERROR]   | [8fdac2a1] Error validating Plex server: (503) service_unavailable; https://plex.tv/api/v2/user <html>
 <head><title>503 Service Temporarily Unavailable</title></head>
 <body>
 <center><h1>503 Service Temporarily Unavailable</h1></center>
 <hr><center>nginx</center>
 </body>
 </html>
 
[2026-04-16 22:09:41,470] [TIMING]  | [8fdac2a1] Loading attribute_config...
[2026-04-16 22:09:41,471] [TIMING]  | [8fdac2a1] Loading collection_config...
[2026-04-16 22:09:41,484] [TIMING]  | [8fdac2a1] Loading overlay_config...
[2026-04-16 22:09:41,489] [TIMING]  | [8fdac2a1] Loading quickstart_root...
[2026-04-16 22:09:41,489] [TIMING]  | [8fdac2a1] Start render_template...
[2026-04-16 22:10:28,793] [INFO]    | [cfd98904] db_cache returned from Plex: 40
[2026-04-16 22:10:28,957] [ERROR]   | [cfd98904] Error validating Plex server: (503) service_unavailable; https://plex.tv/api/v2/user <html>
 <head><title>503 Service Temporarily Unavailable</title></head>
 <body>
 <center><h1>503 Service Temporarily Unavailable</h1></center>
 <hr><center>nginx</center>
 </body>
 </html>
 
[2026-04-16 22:10:28,966] [TIMING]  | [cfd98904] Loading attribute_config...
[2026-04-16 22:10:28,967] [TIMING]  | [cfd98904] Loading collection_config...
[2026-04-16 22:10:28,978] [TIMING]  | [cfd98904] Loading overlay_config...
[2026-04-16 22:10:28,983] [TIMING]  | [cfd98904] Loading quickstart_root...
[2026-04-16 22:10:28,983] [TIMING]  | [cfd98904] Start render_template...
[2026-04-16 22:10:38,713] [DEBUG]   | [cfd98904] Incoming request size: 94.54 KB
```

### Relevant logs (optional)

```shell

```

### Checklist

- [x] I have searched existing issues and this bug has not already been reported
- [x] I can reproduce this issue consistently
- [x] I included the full Support Info output

--- a/modules/helpers.py
+++ b/modules/helpers.py
@@ -99,8 +99,16 @@ def is_valid_aspect_ratio(image, target_ratio="2:3", tolerance=0.01):
 
 def extract_library_name(key):
     """Extracts the actual library name from the key format."""
-    match = re.match(r"(mov|sho)-library_([^-]+(?:-[^-]+)*)-", key)
-    return match.group(2) if match else None
+    if not isinstance(key, str):
+        return None
+    # Capture only the library-id segment between `-library_` and the next
+    # known section marker. This avoids greedy matches when template variable
+    # keys themselves contain hyphens (e.g. `use_South-Eastern Asia`).
+    match = re.match(
+        r"^(?:mov|sho)-library_(.+?)-(?:library$|collection_|template_|attribute_|overlay_|top_level_)",
+        key,
+    )
+    return match.group(1) if match else None
 
 
 def get_pyfiglet_fonts():

--- a/modules/output.py
+++ b/modules/output.py
@@ -1205,6 +1205,20 @@ def build_libraries_section(
                         k: (True if isinstance(v, (bool, str)) and str(v).lower() == "true" else False if isinstance(v, (bool, str)) and str(v).lower() == "false" else v)
                         for k, v in all_children.items()
                     }
+                    # Normalize legacy Region key spelling so final YAML always uses
+                    # the current Kometa key with a hyphen.
+                    legacy_region_keys = {
+                        "use_South Eastern Asia": "use_South-Eastern Asia",
+                        "radarr_add_missing_South Eastern Asia": "radarr_add_missing_South-Eastern Asia",
+                        "sonarr_add_missing_South Eastern Asia": "sonarr_add_missing_South-Eastern Asia",
+                    }
+                    for old_key, new_key in legacy_region_keys.items():
+                        if old_key not in template_vars:
+                            continue
+                        # If both exist, prefer the new key's explicit value.
+                        if new_key not in template_vars:
+                            template_vars[new_key] = template_vars[old_key]
+                        template_vars.pop(old_key, None)
                     if "exclude" in template_vars:
                         exclude_values = _parse_string_list(template_vars.get("exclude"))
                         if exclude_values:

--- a/quickstart.py
+++ b/quickstart.py
@@ -6200,6 +6200,8 @@ def _get_incomplete_resume_runs(limit=25, config_name=None):
             continue
         if is_running and path.name.lower() == "meta.log":
             continue
+        # Prefer ingest-cache mtime to preserve analyzed-run ordering; live
+        # meta.log is refreshed separately below when Kometa is not running.
         mtime = entry.get("mtime")
         if not isinstance(mtime, (int, float)):
             try:
@@ -6213,13 +6215,20 @@ def _get_incomplete_resume_runs(limit=25, config_name=None):
     except Exception:
         live_meta = None
     if live_meta and live_meta.exists() and live_meta.is_file():
-        already_known = any(item[1] == live_meta for item in candidates)
-        if not already_known and not is_running:
+        if not is_running:
             try:
                 live_mtime = live_meta.stat().st_mtime
             except Exception:
                 live_mtime = 0
-            candidates.append((float(live_mtime), live_meta, {"mtime": live_mtime, "run_complete": False}))
+            # Replace any cached candidate for live meta with a fresh mtime candidate.
+            candidates = [item for item in candidates if item[1] != live_meta]
+            live_entry = cache_logs.get(str(live_meta), {}) if isinstance(cache_logs, dict) else {}
+            if not isinstance(live_entry, dict):
+                live_entry = {}
+            if not isinstance(live_entry.get("mtime"), (int, float)):
+                live_entry["mtime"] = live_mtime
+            live_entry.setdefault("run_complete", False)
+            candidates.append((float(live_mtime), live_meta, live_entry))
 
     candidates.sort(key=lambda item: item[0], reverse=True)
     if not candidates:
@@ -6252,6 +6261,7 @@ def _build_latest_incomplete_resume_hint():
         "original_command": latest.get("run_command") or "",
         "suggested_command": latest.get("resume_primary") or "",
         "log_name": latest.get("incomplete_log_name") or "",
+        "log_path": latest.get("incomplete_log_path") or "",
         "config_name": summary_config,
         "context_mismatch": context_mismatch,
         "explanation": latest.get("resume_explanation") if isinstance(latest.get("resume_explanation"), list) else [],

--- a/static/css/styles.css
+++ b/static/css/styles.css
@@ -1850,6 +1850,17 @@ label {
     /* Space text from border */
 }
 
+/* Final page: draw attention to collapsed Kometa section when an update is available */
+.accordion-header.kometa-update-attention {
+    border-left: 4px solid #ffc107 !important;
+    padding-left: 10px !important;
+}
+
+.accordion-button.kometa-update-attention {
+    color: #ffc107 !important;
+    font-weight: 600;
+}
+
 /* Ensure the accordion header has a transparent background */
 .accordion-button:not(.collapsed) {
     background-color: transparent !important;

--- a/static/json/quickstart_collections.json
+++ b/static/json/quickstart_collections.json
@@ -11227,6 +11227,224 @@
         ]
       },
       {
+        "id": "collection_simkl",
+        "label": "Simkl Charts",
+        "url": "https://kometa.wiki/en/nightly/defaults/chart/simkl",
+        "media_types": [
+          "movie",
+          "show"
+        ],
+        "template_variables": [
+          {
+            "key": "collection_section",
+            "label": "Collection Section",
+            "url": "https://kometa.wiki/en/nightly/kometa/guides/order/#kometa-default-collection-sorting",
+            "type": "text_input",
+            "default": "020",
+            "tooltip": "This number determines where this collection appears in the Plex Collections page. Lower numbers/characters based on ascii are placed earlier."
+          },
+          {
+            "key": "limit",
+            "label": "Limit",
+            "type": "text_input",
+            "default": "100",
+            "tooltip": "Maximum number of items to return per Simkl list. Valid range is 1 to 500."
+          },
+          {
+            "key": "use_trending_today",
+            "label": "Simkl Trending Today",
+            "tooltip": "Collection of Simkl's Trending Today list.",
+            "type": "toggle",
+            "default": true
+          },
+          {
+            "key": "radarr_add_missing_trending_today",
+            "label": "Add missing items to Radarr",
+            "tooltip": "Add missing items to Radarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "movie"
+            ]
+          },
+          {
+            "key": "sonarr_add_missing_trending_today",
+            "label": "Add missing items to Sonarr",
+            "tooltip": "Add missing items to Sonarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "show"
+            ]
+          },
+          {
+            "key": "use_trending_week",
+            "label": "Simkl Trending This Week",
+            "tooltip": "Collection of Simkl's Trending This Week list.",
+            "type": "toggle",
+            "default": true
+          },
+          {
+            "key": "radarr_add_missing_trending_week",
+            "label": "Add missing items to Radarr",
+            "tooltip": "Add missing items to Radarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "movie"
+            ]
+          },
+          {
+            "key": "sonarr_add_missing_trending_week",
+            "label": "Add missing items to Sonarr",
+            "tooltip": "Add missing items to Sonarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "show"
+            ]
+          },
+          {
+            "key": "use_trending_month",
+            "label": "Simkl Trending This Month",
+            "tooltip": "Collection of Simkl's Trending This Month list.",
+            "type": "toggle",
+            "default": true
+          },
+          {
+            "key": "radarr_add_missing_trending_month",
+            "label": "Add missing items to Radarr",
+            "tooltip": "Add missing items to Radarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "movie"
+            ]
+          },
+          {
+            "key": "sonarr_add_missing_trending_month",
+            "label": "Add missing items to Sonarr",
+            "tooltip": "Add missing items to Sonarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "show"
+            ]
+          },
+          {
+            "key": "use_dvd",
+            "label": "Simkl DVD",
+            "tooltip": "Collection of Simkl's Recently Released on DVD list.",
+            "type": "toggle",
+            "default": true
+          },
+          {
+            "key": "radarr_add_missing_dvd",
+            "label": "Add missing items to Radarr",
+            "tooltip": "Add missing items to Radarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "movie"
+            ]
+          },
+          {
+            "key": "sonarr_add_missing_dvd",
+            "label": "Add missing items to Sonarr",
+            "tooltip": "Add missing items to Sonarr can generate a lot of internet traffic and disk space issues depending on the number of items missing in your library. <br> <b>Use with caution.</b> ",
+            "tooltip_variant": "danger",
+            "type": "toggle",
+            "media_types": [
+              "show"
+            ]
+          },
+          {
+            "key": "style",
+            "label": "Style",
+            "type": "select",
+            "tooltip": "Controls the visual theme of the collections created.",
+            "default": "color",
+            "options": [
+              "color",
+              "white"
+            ]
+          },
+          {
+            "key": "collection_mode",
+            "label": "Collection Mode",
+            "type": "select",
+            "tooltip": "Controls the collection mode of all collections in a Defaults File.<br><br> Default is <b>Library Default</b> ",
+            "default": "default",
+            "options": [
+              {
+                "value": "default",
+                "label": "Library Default"
+              },
+              {
+                "value": "hide",
+                "label": "Hide Collection"
+              },
+              {
+                "value": "hide_items",
+                "label": "Hide Items in this Collection"
+              },
+              {
+                "value": "show_items",
+                "label": "Show this Collection and its Items"
+              }
+            ]
+          },
+          {
+            "key": "visible_home",
+            "label": "Visible Home",
+            "type": "toggle",
+            "tooltip": "Changes collection visible on Home Tab (Only works with Plex Pass)",
+            "default": false
+          },
+          {
+            "key": "visible_library",
+            "label": "Visible Library",
+            "type": "toggle",
+            "tooltip": "Changes collection visible on Library Recommended Tab (Only works with Plex Pass)",
+            "default": false
+          },
+          {
+            "key": "visible_shared",
+            "label": "Visible Shared",
+            "type": "toggle",
+            "tooltip": "Changes collection visible on Shared Users' Home Tab (Only works with Plex Pass)",
+            "default": false
+          },
+          {
+            "key": "collection_order",
+            "label": "Collection Order",
+            "type": "select",
+            "tooltip": "Changes the Collection Order for all collections in a Defaults File.<br><br> Default is <b>Custom (sorted in a custom order)</b> ",
+            "default": "custom",
+            "options": [
+              {
+                "value": "custom",
+                "label": "Custom Order",
+                "media_types": [
+                  "movie",
+                  "show",
+                  "episode"
+                ]
+              },
+              {
+                "value": "alpha",
+                "label": "Alphabetical Order",
+                "media_types": [
+                  "movie",
+                  "show",
+                  "episode"
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      {
         "id": "collection_anilist",
         "label": "AniList Charts",
         "url": "https://kometa.wiki/en/nightly/defaults/chart/anilist",
@@ -20840,7 +21058,7 @@
             "default": true
           },
           {
-            "key": "use_South Eastern Asia",
+            "key": "use_South-Eastern Asia",
             "label": "South-Eastern Asia",
             "tooltip": "Collection of Movies that have been filmed in South-Eastern Asia.",
             "type": "toggle",

--- a/static/local-js/900-final.js
+++ b/static/local-js/900-final.js
@@ -84,6 +84,9 @@ $(document).ready(function () {
   const headerGridProgress = document.getElementById('header-style-grid-progress')
   const headerGridProgressBar = headerGridProgress ? headerGridProgress.querySelector('.progress-bar') : null
   const headerStyleLabel = document.getElementById('header-style-label')
+  const kometaActionsHeading = document.getElementById('kometa-actions-heading')
+  const kometaActionsCollapse = document.getElementById('kometa-actions-collapse')
+  const kometaActionsToggle = document.getElementById('kometa-actions-toggle')
 
   function readMetaFlag (id, datasetKey, attrKey) {
     const el = document.getElementById(id)
@@ -775,6 +778,19 @@ $(document).ready(function () {
     })
   }
 
+  function syncKometaUpdateAttention () {
+    if (!kometaActionsHeading || !kometaActionsToggle) return
+    const isCollapsed = kometaActionsToggle.classList.contains('collapsed')
+    const needsAttention = KOMETA_UPDATE_AVAILABLE && isCollapsed
+    kometaActionsHeading.classList.toggle('kometa-update-attention', needsAttention)
+    kometaActionsToggle.classList.toggle('kometa-update-attention', needsAttention)
+  }
+
+  if (kometaActionsCollapse) {
+    kometaActionsCollapse.addEventListener('shown.bs.collapse', syncKometaUpdateAttention)
+    kometaActionsCollapse.addEventListener('hidden.bs.collapse', syncKometaUpdateAttention)
+  }
+
   function showCopyButtonSuccess (iconSelector, textSelector) {
     const $icon = $(iconSelector)
     const $text = $(textSelector)
@@ -1113,6 +1129,7 @@ $(document).ready(function () {
     if ($updateKometaBtn.length) {
       $updateKometaBtn.html(getUpdateButtonLabel())
     }
+    syncKometaUpdateAttention()
   }
 
   function callUpdateKometa () {

--- a/static/local-js/eventHandler.js
+++ b/static/local-js/eventHandler.js
@@ -386,6 +386,18 @@ const EventHandler = {
   },
 
   /**
+   * Returns true if an accordion body has at least one enabled collection toggle selected.
+   * Returns false when collection toggles exist but none are selected.
+   * Returns null when no collection toggles are present.
+   */
+  hasCheckedTemplateGroupToggle: function (accordionBody) {
+    if (!accordionBody) return null
+    const toggles = Array.from(accordionBody.querySelectorAll("input[type='checkbox'][data-template-group]"))
+    if (!toggles.length) return null
+    return toggles.some(toggle => toggle.checked)
+  },
+
+  /**
    * Update accordion highlights when selections change
    */
   updateAccordionHighlights: function () {
@@ -447,6 +459,14 @@ const EventHandler = {
             return toggle?.checked
           })
         }
+      }
+
+      // Collection accordions should not stay highlighted from child values/history
+      // when every parent collection toggle is off.
+      const anyTemplateGroupChecked = EventHandler.hasCheckedTemplateGroupToggle(accordionBody)
+      if (anyTemplateGroupChecked === false) {
+        isCheckedOrSelected = false
+        hasValue = false
       }
 
       if (isCheckedOrSelected || hasValue) {
@@ -562,7 +582,11 @@ const EventHandler = {
       '.list-group li'
     ) !== null
 
-    if (!hasSelections) {
+    // If this accordion has collection toggles and none are enabled, force no highlight.
+    const anyTemplateGroupChecked = EventHandler.hasCheckedTemplateGroupToggle(accordionBody)
+    const effectiveSelections = (anyTemplateGroupChecked === false) ? false : hasSelections
+
+    if (!effectiveSelections) {
       element.classList.remove('selected')
     }
 

--- a/templates/900-final.html
+++ b/templates/900-final.html
@@ -482,7 +482,7 @@
     <div class="accordion mt-3" id="kometa-actions-accordion" style="max-width: 1200px; width: 100%; margin: 0 auto;">
       <div class="accordion-item">
         <h2 class="accordion-header" id="kometa-actions-heading">
-          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+          <button id="kometa-actions-toggle" class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
             data-bs-target="#kometa-actions-collapse" aria-expanded="false" aria-controls="kometa-actions-collapse">
             Kometa Install / Update
           </button>
@@ -566,6 +566,11 @@
                       | Config in log: <code>{{ incomplete_resume_hint.config_name }}</code>
                       {% endif %}
                     </div>
+                    {% if incomplete_resume_hint.log_path %}
+                    <div class="small text-muted mb-1 text-break">
+                      Source path: <code>{{ incomplete_resume_hint.log_path }}</code>
+                    </div>
+                    {% endif %}
                     {% if incomplete_resume_hint.context_mismatch %}
                     <div class="small text-danger mb-1">
                       Warning: this incomplete run was recorded under a different config name than the one currently loaded.

